collect_app/src/main/java/org/odk/collect/android/activities/FormFillingActivity.java

```diff
@@ -1035,7 +1034,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
 
             case R.id.menu_save:
                 // don't exit
-                saveForm(false, InstancesDaoHelper.isInstanceComplete(getFormController()), null, true);
+                saveForm(false, false, null, true);
```

Comment:
This was a super easy change as the tests already describe the desired behaviour. I just had to get rid of the extra untested logic